src/libraries/System.Private.CoreLib/src/System/Globalization/Ordinal.cs

```diff
@@ -78,39 +78,73 @@ internal static int CompareStringIgnoreCaseNonAscii(ref char strA, int lengthA,
             return OrdinalCasing.CompareStringIgnoreCase(ref strA, lengthA, ref strB, lengthB);
         }
 
-        private static bool EqualsIgnoreCase_Vector128(ref char charA, ref char charB, int length)
+        private static bool EqualsIgnoreCase_Vector<TVector>(ref char charA, ref char charB, int length)
```

Comment:
Not specific to just this PR, but would the expectation be that the different `ISimdVector` impl variants are inlined into the caller?
Or should we be spamming them with aggressive inlining / counting on PGO to decide to avoid the extra calls into the correct helper?

Reply:
I think it depends on a case? in this case the original function was not inlineable too